burn-import/src/pytorch/remapping.rs

```diff
@@ -0,0 +1,170 @@
+use std::collections::HashMap;
+
+use super::{converter::ModuleType, Converter};
+use candle_core::Tensor as CandleTensor;
+
+use lazy_static::lazy_static;
+use regex::Regex;
+
+lazy_static! {
```

Comment:
I would use `once_cell` instead of `lazy_static!` because it is part of `std` now, https://doc.rust-lang.org/std/cell/struct.OnceCell.html, so we reduce the number of dependencies